pages/_app.tsx

```diff
@@ -11,9 +12,20 @@ import CENNZApiProvider from "../providers/CENNZApiProvider";
 import SupportedWalletProvider from "../providers/SupportedWalletProvider";
 import DappModuleProvider from "../providers/DappModuleProvider";
 import Web3AccountsProvider from "../providers/Web3AccountsProvider";
+import Switch from "../components/AppSwitch";
+import Wallet from "../components/Wallet";
+import SupportedAssetsProvider from "../providers/SupportedAssetsProvider";
+import BlockchainProvider from "../providers/BlockchainProvider";
 
 function MyApp({ Component, pageProps }: AppProps) {
 	const router = useRouter();
+	const [location, setLocation] = useState<string>();
+
+	useEffect(() => {
+		if (location !== undefined) router.push(`/${location}`);
+		//eslint-disable-next-line
+	}, [location]);
```

Comment:
`//eslint-disable-next-line` because we cannot add `router` in the dependency array?